Mathlib/SetTheory/Cardinal/Aleph.lean

```diff
@@ -91,8 +96,159 @@ def isInitialIso : {x // IsInitial x} ≃o Cardinal where
   right_inv x := card_ord x
   map_rel_iff' {a _} := a.2.card_le_card
 
--- TODO: define `omega` as the enumerator function of `IsInitial`, and redefine
--- `aleph x = (omega x).card`.
+/-- The `omega'` function gives the initial ordinals listed by their ordinal index. `omega' n = n`,
+`omega' ω = ω`, `aleph' (ω + 1) = ω₁`, etc.
+
+For the more common omega function skipping over finite ordinals, see `Ordinal.omega`. -/
+def omega' : Ordinal.{u} ↪o Ordinal.{u} where
```

Comment:
I'm wondering if we should change the naming convention, since we want to move away from primed names. What do we think of `preOmega` (somewhat similar to `IsPreconnected`)?

Whatever we choose, it should be consistent with `aleph'`, and can wait for a follow-up PR if you prefer.

Reply:
That name is fine, if for no other reason that I have no better suggestions. I don't like the idea of introducing `omega'` and then immediately deprecating it though, so I'll do the rename for `aleph'` first.

Reply:
#18099